canonical-websites/www.ubuntu.com
# Build the Tackling the IoT takeover

## Done
Removed the session selecting JS. Replaced the existing two takeovers with the copy and design. Added a simple animation to the piggy bank. Changed the takeover styling to be inline so that we only need to simply change the include to see old takeovers.

**Do not merge**: 
1. Still awaiting a link for the whitepaper.
2. The right-side suru image provided does not match the design.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that the copy matches [the copy doc](https://docs.google.com/document/d/1gcvBmsTPoJFdLB4p8A92tbjWVHQEYQOlZugFrBBYnek/edit#heading=h.meo0azuz09li)
- Check that it matches [the design](https://github.com/canonical-websites/www.ubuntu.com/issues/3892#issuecomment-414363647)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/3894
